subsys/bluetooth/common/Kconfig

```diff
@@ -77,10 +77,21 @@ config BT_BUF_ACL_RX_SIZE
 	  In a Controller only build this will determine the maximum ACL size
 	  that the Controller will send to the Host.
 
+config BT_BUF_ACL_RX_COUNT_EXTRA
+	int "Number of extra incoming ACL data buffers"
+	default 0
+	range 0 65535
+	help
+	  Number of incoming extra ACL data buffers sent from the Controller to
+	  the Host.
+
+	  By default, the number of incoming ACL data buffers is equal to
+	  CONFIG_BT_MAX_CONN + 1.
```

Comment:
```suggestion
	  By default, the number of incoming ACL data buffers is equal to
	  CONFIG_BT_MAX_CONN + 1. The additional buffer is required because
	  of the way we re-assemble ACL packets into L2CAP packets.
```

Reply:
We plan on adding more documentation on that process on a following PR, I think adding that here is unnecessary for now. Especially since `CONFIG_BT_BUF_ACL_RX_COUNT` documentation is still there for now.